src/openforms/formio/migration_converters.py

```diff
@@ -76,6 +78,48 @@ def set_openforms_datasrc(component: Component) -> bool:
     return True
 
 
+def fix_column_sizes(component: Component) -> bool:
+    component = cast(ColumnsComponent, component)
+
+    changed = False
+    for column in component.get("columns", []):
+        size = column.get("size", "6")
+        size_mobile = column.get("sizeMobile")
+
+        size_ok = isinstance(size, int)
+        size_mobile_ok = size_mobile is None or isinstance(size_mobile, int)
+
+        if size_ok and size_mobile_ok:
+            continue
+
+        changed = True
+        if not size_ok:
+            try:
+                column["size"] = int(size)
+            except (TypeError, ValueError):
+                column["size"] = 6
```

Comment:
Would be nice to see what was set initially in this case. I'm wondering what could have happened to have an non integer value set. I guess the bin script in the other PR will reveal

Reply:
I considered logging, but nobody is actually going to look at that and do anything with it. The detection script can we wired up as an upgrade check so that upgrading is blocked until those issues are resolved, but in this case it would cause run-time issues anyway so I opted to silently fix it instead.

I don't see a way without excessive effort to nicely package and format the information and get it delivered to the people actually building/maintaining the forms themselves.